CHANGELOG.md

```diff
@@ -35,8 +35,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 
 - Handling of device unsetting the ADR bit in uplink, after ADR has been started.
-- Invalid `oauth-server-address` in CLI config generated by `use` command when config file is already present.
-- Properly creating ABP devices from the CLI when reading devices from stdin.
+- Data rate 4 in version `1.0.2-a` of AU915.
+- Incorrect `TxOffset` values used by Network Server in some bands.
```

Comment:
Is this a conflict resolution error?

Reply:
No, that's a long standing issue apparently. It was all a mutable mess, so each downgrade could potentially mutate other references existing and future.